server/lib/packets/packet.go

```diff
@@ -92,16 +103,39 @@ func (pkt *RecvPacket) Process(t Transport) {
 	}
 }
 
+func Actor2ProtoError(aerr *yue.ActorError) *proto.Err {
```

Comment:
__[gometalinter]__
* [golint] exported function Actor2ProtoError should have comment or be unexported
* [gotype] undeclared name: yue
* [gotype] undeclared name: proto